autoconnect/src/main.rs

```diff
@@ -79,9 +82,24 @@ async fn main() -> Result<()> {
         "Starting autoconnect on port {} (router_port: {})",
         port, router_port
     );
-    HttpServer::new(move || build_app!(app_state))
-        .bind(("0.0.0.0", port))?
-        .bind(("0.0.0.0", router_port))?
+
+    let app_state2 = app_state.clone();
```

Comment:
Even though it's essentially a temporary value, kinda wondering if we should name it `router_state` or something a bit more intentional.